### ISSUE TITLE
crash on ios

### ISSUE THREAD

**Issue by TurboTu** [crykn/libgdx-screenmanager#29]
Hi, I want to ask if this can used on iOS. 

> IDE: Android Studio Giraffe | 2022.3.1 Patch 3
> LibGDX: 1.12.1
> libgdx-screenmanager: 0.7.0
> iOS: 17.2.1

first, I create a MyGame class and initial in create:
```
public class MyGame extends ManagedGame<ManagedScreen, ScreenTransition> {
    ...
   @Override
    public void create() {
        super.create();
        batch = new SpriteBatch();
        ...
        this.screenManager.pushScreen(new MainMenuScreen(this), new BlendingTransition(batch, 1F));
    }
}
```
then I create the MainMenuScreen class:
```
public class MainMenuScreen extends ManagedScreen {
    ...
}
```
this code works on Android but when I use RoboVM to create the ipa, it crashed on iOS.
Is there any special operations needed?
And if I don't use this lib, the whole app packed into ipa can work on iOS

**Comment by crykn** [comment]
I've never used libGDX with iOS, so I don't have any first-hand experience with this. In theory however, this library should absolutely work with iOS. Did the crash message provide some insight on what exactly caused the crash?

**Comment by TurboTu** [comment]
Because my IDE is Android Studio, I cannot get the debug info, but I can get something from iOS console print messages.Now I'm out and didn't take my Mac, when I'm back, I can collect the infomation and send it here. I remembered it was about someting like "scene"

**Comment by crykn** [comment]
Fixed by 0bb0df6aa683b24f40df6e23b70828593e975959.
